site.js

```diff
@@ -139,16 +140,22 @@ function unzoomImage() {
   });
 }
 
+function handleLoadedClues(jsonData) {
+  var jsonObj = jsonData[0];
+  clues.push({name: 'population', value: jsonObj.population});
+  clues.push({name: 'region', value: jsonObj.region});
+  clues.push({name: 'currencies', value: jsonObj.join(', ')});
+  clues.push({name: 'borders', value: jsonObj.join(', ')});
+  clues.push({name: 'nativeName', value: jsonObj.nativeName});
```

Comment:
You need to reset `clues` to an empty array. Otherwise, you're just going to get 5* `hintsUsed` clues later on, and as many as `(hintsUsed - 1) * 5` will be incorrect.